xcode/Subconscious/SubconsciousTests/Tests_Subtext.swift

```diff
@@ -269,6 +269,54 @@ class Tests_Subtext: XCTestCase {
         )
     }
     
+    func testAbsoluteSlashlinkParsing1() throws {
+        let markup = "@petname/slashlink at the beginning."
+        let dom = Subtext.parse(markup: markup)
+        
+        guard case let .slashlink(slashlink) = dom.blocks[0].inline[0] else {
+            XCTFail("Expected slashlink")
+            return
+        }
+        
+        XCTAssertEqual(
+            String(describing: slashlink),
+            "@petname/slashlink",
+            "Slashlink parses successfully"
+        )
+    }
+
+    func testAbsoluteSlashlinkParsing2() throws {
+        let markup = "@PETNAME/deep/slashlink at the beginning."
+        let dom = Subtext.parse(markup: markup)
+        
+        guard case let .slashlink(slashlink) = dom.blocks[0].inline[0] else {
+            XCTFail("Expected slashlink")
+            return
+        }
+        
+        XCTAssertEqual(
+            String(describing: slashlink),
+            "@PETNAME/deep/slashlink",
+            "Slashlink parses successfully"
+        )
+    }
+
+    func testAbsoluteSlashlinkParsingDoesNotValidateSlashlink() throws {
+        let markup = "@PETNAME/deep//slashlink/that-is-technically-invalid at the beginning."
+        let dom = Subtext.parse(markup: markup)
+        
+        guard case let .slashlink(slashlink) = dom.blocks[0].inline[0] else {
+            XCTFail("Expected slashlink")
+            return
+        }
+        
+        XCTAssertEqual(
+            String(describing: slashlink),
+            "@PETNAME/deep//slashlink/that-is-technically-invalid",
+            "Slashlinks do not parse for valididty of body"
+        )
+    }
```

Comment:
Could use a test case that looks for an `@petname` that isn't at the start of the string, i.e.

```swift
let markup = "What about @petname/nested/link in the middle."
```